### PR TITLE
Title: implement user sign up feature

### PR DESCRIPTION
**Pull request by valentineezeh**
### What does this PR do?
- This PR configure the base app set up and implement the sign up feature

### Description of task to be completed:
- Set up and configure base app
- Set up and configure database
- Configure and set-up environmental variables
- Implement table creation by configuring and setting up migrations
- Implement sign up controller
- Implement user input validations
- Write tests
signup_feature

### How can this  task be manually tested?
- Clone repo
- Install all dependencies in requirements.txt by running `pip install -r requirements.txt`
- Run migrations `python manage.py db migrate`
- Run server by running `flask run` or `python run.py`
- Visit `localhost:5000/api/auth/signup` on postman and input the required field to sign up a user.
signup_feature

### Link to Trello task card
[card_id: NidxIIrA](https://trello.com/c/NidxIIrA/1-as-a-user-i-should-be-able-to-create-an-account-with-my-email-phone-number-and-password#)

### CODE REVIEW

File: controllers/Sign_up.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
+from flask import request, jsonify
+from flask.views import MethodView
```
Comment (obinnaeye) [inline controllers/Sign_up.py]: This line is redundant, not used in the code

File: app/__init__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# app/__init__.py
+from flask import Flask
+from flask_api import FlaskAPI
+from flask_sqlalchemy import SQLAlchemy
+
+from flask import request, jsonify, abort
```
Comment (obinnaeye) [inline app/__init__.py]: You don't seem to be using any of these in the file

Comment (valentineezeh) [reply app/__init__.py]: I have remove:
`from flask_api import FlaskAPI` and `from flask import request, jsonify, abort`
But I am using `from flask import Flask` and `from flask_sqlalchemy import SQLAlchemy` for the app set up and db.

File: controllers/Sign_up.py (hunk 1 above)
Comment (valentineezeh) [reply controllers/Sign_up.py]: I have removed `from flask.view import MethodView`.

**Pull request merged**